packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js

```diff
@@ -193,7 +193,12 @@ async function parseExportNamesInto(
           if (typeof source !== 'string') {
             throw new Error('Expected the transformed source to be a string.');
           }
-          parseExportNamesInto(source, names, url, defaultTransformSource);
+          await parseExportNamesInto(
```

Comment:
At line 173, this was a async function and await was missing, this should be added here